### PR TITLE
Add Fedora Ontology

### PR DESCRIPTION
**Pull request by acoburn**

### CODE REVIEW

File: repository.owl [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
```
Comment (awoods) [inline repository.owl]: Would an `application/owl+xml` serialization make sense for better HTML/XSLT rendering?
https://www.w3.org/TR/owl2-xml-serialization/

Comment (acoburn) [inline repository.owl]: I have no interest in writing XML (I find it difficult to read and difficult to maintain), but if you'd like to convert this document, be my guest.

Comment (awoods) [reply repository.owl]: What is the mime-type of this current serialization?

Comment (acoburn) [inline repository.owl]: it is TURTLE: `text/turtle`

Comment (awoods) [reply repository.owl]: Would a `.ttl` extension make more sense?

Comment (acoburn) [inline repository.owl]: using `.ttl` would be fine. I've just changed it in a new commit.

**Comment by awoods** [comment]
:+1:

**Comment by awoods** [comment]
@ruebot : you should now be good going forward.

**Pull request merged**